File: C/Listas/Lista 1/ex14.c
//Lê um valor de hora e informa quantos minutos se passaram desde o início do dia.
#include <stdio.h>

int main(){
    int hora;
    printf("Insira o valor da hora: ");
    scanf("%d", &hora);
    
    printf("Conforme o valor de %d horas inserido, o total de minutos que se passaram desde o início do dia será de %d minutos!", hora, hora*60);
    return 0;
}

File: Java/extra/Lista 01/src/ex14.java
// Lê um valor de hora e informa quantos minutos se passaram desde o início do dia.

import java.util.Scanner;

public class ex14 {
    static int relogio(int a) {
        int x = a * 60;
        return x;
    }

    public static void main(String[] args) {
        int horas;
        System.out.println("Digite a hora que deseja usar para calcular os minutos: ");
        try (Scanner leitor = new Scanner(System.in)) {
            horas = leitor.nextInt();
        }

        System.out.format("\nJá se passaram exatos %d minutos desde o início do dia", relogio(horas));
    }
}
